roles/confluent.variables/defaults/main.yml

```diff
@@ -39,6 +39,9 @@ open_file_limit: 500000
 ### Boolean to install commercially licensed confluent-server instead of community version: confluent-kafka
 confluent_server_enabled: true
```

Comment:
```suggestion
### Boolean to install commercially licensed confluent-server instead of community version: confluent-kafka.
confluent_server_enabled: "{{ not confluent_community}}"
```
WDYT about setting these variables based off of the confluent_community variable? And then having assertions on them as well?